modules/tinymce/src/themes/silver/main/ts/ui/promotion/Promotion.ts

```diff
@@ -0,0 +1,30 @@
+import { AlloySpec, SimpleSpec } from '@ephox/alloy';
+
+// TODO final text and link to be decided in https://ephocks.atlassian.net/browse/TINY-8954
+const promotionMessage = 'Try Premium!';
```

Comment:
According to the edits made to the Jira overnight, this should say `Upgrade` for now 🤷